src/test/java/igym/repositories/UserRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package igym.repositories;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import igym.entities.UserEntity;
+
+@SpringBootTest
+@ActiveProfiles("test")
+public class UserRepositoryTest {
+
+    @Autowired 
+    UserRepository repository;
+
+    @Test
+    @DisplayName("Testing persisting User in the DB and getting it again by ID")
+    public void saveUserTest() {
+        String name1 = "John Snow";
+        UserEntity user1 = new UserEntity(name1);
+        UserEntity savedUser = repository.save(user1);
+        assertEquals(user1.getName(), savedUser.getName());
```

Comment:
Makes the code more readible

```suggestion
        UserEntity savedUser = repository.save(new UserEntity("John Snow"));
        assertEquals("John Snow", savedUser.getName());
```